src/native/managed/cdacreader/tests/MockDescriptors.cs

```diff
@@ -557,11 +654,16 @@ internal TargetPointer AddThread(uint id, TargetNUInt osId)
             _builder.AddHeapFragment(thread);
 
             // Add exception info for the thread
-            MockMemorySpace.HeapFragment exceptionInfo = _allocator.Allocate(Types[DataType.ExceptionInfo].Size.Value, "ExceptionInfo");
-            _builder.AddHeapFragment(exceptionInfo);
+            //MockMemorySpace.HeapFragment exceptionInfo = _allocator.Allocate(Types[DataType.ExceptionInfo].Size.Value, "ExceptionInfo");
+            //_builder.AddHeapFragment(exceptionInfo);
+            TargetPointer exceptionInfoAddress = default;
+            if (UseFunclets)
+                throw new NotImplementedException("todo for funclets: allocate and ExceptionInfo");
+            else
+                exceptionInfoAddress = thread.Address + Types[DataType.ExceptionInfo].Size.Value;
             helpers.WritePointer(
                 data.Slice(typeInfo.Fields[nameof(Data.Thread.ExceptionTracker)].Offset),
-                exceptionInfo.Address);
+                exceptionInfoAddress);
```

Comment:
@elinor-fung this is the bit that fixes the regression that I introduced when I changed the bump allocator to align allocations at 16-byte boundaries by default.  The Thread and ExceptionInfo allocations ended up with a gap between them, and the contract expected the second field of ExceptionInfo to be right in that hole.